src/expressions/operator-expr.md

```diff
@@ -483,7 +483,8 @@ assert_eq!(values[1], 3);
 `*const T` / `*mut T` can be cast to `*const U` / `*mut U` with the following behavior:
 
 - If `T` is unsized and `U` is sized, the cast discards all metadata that
-  completes the wide pointer `T` and produces a thin base pointer `U`.
+  completes the wide pointer `T` and produces a thin pointer `U` to the
+  value of the unsized pointer.
```

Comment:
I'm struggling to find a phrasing here for to describe what "value" is which applies to all unsized pointers and also conveys *what exactly it points to*. In the reference for trait objects "instance" is used, but that would be a bit unorthodox for slices.